packages/@react-spectrum/labeledvalue/test/LabeledValue.test.js

```diff
@@ -275,6 +275,33 @@ describe('LabeledValue', function () {
     expect(staticField).toHaveTextContent('10 – 20');
   });
 
+  it('renders correctly with ReactNode value', function () {
+    let {getByTestId} = render(
+      <LabeledValue
+        data-testid="test-id"
+        label="Field label"
+        value={<a href="https://test.com">test</a>} />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).toHaveTextContent('test');
+    expect(
+      within(staticField).getByRole('link', {name: 'test'})
+    ).toBeInTheDocument();
+  });
+
+  it('throws when an editable value is provided', async function () {
+    await waitFor(() => { 
```

Comment:
this is going to loop over render a bunch of times potentially, which is probably why you're getting the "unmounted while working" or "shouldn't already be working"

This is probably a good StackOverflow to reference on how to fix this test https://stackoverflow.com/questions/66328549/testing-an-error-thrown-by-a-react-component-using-testing-library-and-jest

the short of it being that you have to catch the console.error in addition to the thrown error since our tests fail if there is a console error